guide/interactions/context-menus.md

```diff
@@ -15,7 +15,7 @@ const { ContextMenuCommandBuilder } = require('@discordjs/builders');
 
```

Comment:
idk if thats the right import tho
```suggestion
const { ApplicationCommandOptionTypes } = require('discord-api-types/v9')

```

Reply:
https://discord.js.org/#/docs/discord.js/stable/typedef/ApplicationCommandOptionType

Can't this be used instead here :thinking:

Reply:
builders exports a ContextMenuType enum right? just use that?

Reply:
> https://discord.js.org/#/docs/discord.js/stable/typedef/ApplicationCommandOptionType
> 
> Can't this be used instead here 🤔

No, I have mistaken this for options, but its an ApplicationCommand type.